avm/res/api-management/service/version.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
-    "version": "0.3",
+    "version": "0.3.1",
```

Comment:
Hi @AlexanderSehr,
Thanks for letting me know. I have fixed that and just updated the minor version since this would be a new feature. Please let me know if anything else is needed to move this along. Thanks!